public/app/features/search/page/components/columns.tsx

```diff
@@ -127,20 +119,26 @@ export const generateColumns = (
         classNames += ' ' + styles.missingTitleText;
       }
       return (
-        <a {...p.cellProps} href={p.userProps.href} onClick={p.userProps.onClick} className={classNames} title={name}>
-          {name}
-        </a>
+        <div className={styles.cell} {...p.cellProps}>
+          {!response.isItemLoaded(p.row.index) ? (
+            <Skeleton width={200} />
+          ) : (
+            <a href={p.userProps.href} onClick={p.userProps.onClick} className={classNames} title={name}>
+              {name}
+            </a>
+          )}
+        </div>
       );
     },
     id: `column-name`,
     field: access.name!,
-    Header: () => <div className={styles.headerNameStyle}>{t('search.results-table.name-header', 'Name')}</div>,
+    Header: () => <div>{t('search.results-table.name-header', 'Name')}</div>,
     width,
   });
   availableWidth -= width;
 
   width = TYPE_COLUMN_WIDTH;
```

Comment:
I think we can simplify this width logic considerably more. 

Previously this was laying things out with pixels (using the container width that the auto-sizer gets), but now this custom flex layout uses relative values, so I don't think we need to do this whole `availableWidth -= width` stuff any more? Each column can just have a static number that'll work out (like how it works in the tree)

Reply:
@joshhunt one extra thing it does do is remove columns that don't have enough width remaining. this is helpful for smaller viewports (e.g. mobile). in that case the `tags`/`location` columns are completely removed. so whilst we can simplify, think we can hold off doing that until the unification work between this table and the `DashboardsTree` table. wdyt? 🤔